### DISCUSSION TITLE
Sync to localhost:5341 from Service Worker (Windows service)

### DISCUSSION THREAD

**Discussion by clabnet** [datalust/seq-tickets#2278] [Help]
In a windows service .net 8 with this configuration 
```
Log.Logger = new LoggerConfiguration()
    .MinimumLevel.Information()
    .WriteTo.Seq("http://localhost:5341")
    .CreateLogger();
```
the messages are not shown into web dashboard. 
Any suggestion to configure it to sync directly in realtime to Seq dashboard ? 
Thank you in advance.

**Comment by nblumhardt** [comment]
Hi Claudio,

The Ingestion Troubleshooting page has a few suggestions for diagnosing this issue:

https://docs.datalust.co/docs/ingestion-troubleshooting

If that doesn't help, posting some more details around how logging is used in the service would help (i.e. an example logging statement for a message that's expected but missing from Seq).
